beam-sqlite/test/Database/Beam/Sqlite/Test/InsertOnConflictReturning.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE DerivingStrategies #-}
+
+{-
+    This module is based on the bug reproducer from issue #773
+    https://github.com/haskell-beam/beam/issues/773
+-}
+module Database.Beam.Sqlite.Test.InsertOnConflictReturning (tests) where
+
+import Data.Int (Int32)
+import Data.Text (Text)
+import Data.Time (Day (..), LocalTime, UTCTime (..), fromGregorian, getCurrentTime, secondsToDiffTime)
```

Comment:
I don't think this import is used, is it?

Reply:
Indeed! Why did my editor not tell me?